TeamCode/src/test/java/org/firstinspires/ftc/teamcode/auton/smc/StateMachineOpModeTest.java

```diff
@@ -0,0 +1,49 @@
+package org.firstinspires.ftc.teamcode.auton.smc;
+
+import org.junit.Test;
+import org.junit.Assert;
+
+import org.firstinspires.ftc.teamcode.auton.smc.states.*;
+import org.firstinspires.ftc.teamcode.auton.smc.StateMachineOpMode;
+
+import java.util.HashMap;
+
+public class StateMachineOpModeTest {
+    @Test
+    public void currentStateIsNonexistentInLoop() {
+        Exception exception = Assert.assertThrows(RuntimeException.class, () -> {
+            StateMachineOpMode stateMachine = new StateMachineOpMode();
+            stateMachine.init();
+            stateMachine.states = new HashMap<>();
+            stateMachine.loop();
+        });
+        System.out.println("Exception message: " + exception.getMessage());
```

Comment:
You generally only want to print stuff to `stdout` (the console) when things fail, to decrease verbosity. Consider removing this `System.out.println(...)`.

Reply:
oops... I meant to delete that